### PR TITLE
add `<taxonomy>` and `<category>` to att.datcat (issue #2419 )

### LINKED ISSUE #2419: add `<taxonomy>` and `<category>` to att.datcat
This is a request coming from the Lexical Resources Summit 2023 (DARIAH, Berlin), convened by @ttasovac and @laurentromary . The immediate context is the TEI Lex0 customisation but @JessedeDoes suggests that the request could also be helpful in the ParlaMint project.

Essence: it would be most useful to be able to use the [datcat attributes](https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-att.datcat.html) for taxonomies, of any sort (datcat atts are not only a grammatical device, any longer). For that purpose, the `<taxonomy>` and `<category>` elements should be members of att.datcat.

When discussing the initial bundle of elements for the re-written datcat, a few months ago, we decided to start small and expand when there is a need. Now, the need comes from two well-established projects, and there is a good chance that the addition is going to be useful elsewhere as well.

Also, we said that, since taxonomies may use `<equiv>`, we'd see if a genuine need for the attribute class exists. Neither the ODD for Lex-0 nor the ODD for ParlaMint uses the tagdocs module. I think that this may qualify as genuine need, because requiring these carefully crafted ODDs to use the tagdocs module only for the sake of `equiv/@url` as the mechanism for aligning with external taxonomies is definitely far-fetched.

We're going to add a PR to this ticket, with some examples added to (at least) the att.datcat spec.

### PR DESCRIPTION
**Pull request by bansp**
This PR extends the description in the att.datcat spec and adds `taxonomy` and `category` elements to that class.
The description adds to the the linguistic picture but also shows the usage of the attributes elsewhere, in usage labels in dictionaries (the `usg` element).

Some references to the example dictionary are missing in the initial version, pending beautification.

The result can be seen at https://jenkins-paderborn.tei-c.org/view/LingSIG/job/TEIP5-LingSIG-tests/lastSuccessfulBuild/artifact/P5/release/doc/tei-p5-doc/en/html/ref-att.datcat.html

### CODE REVIEW

File: P5/Source/Specs/att.datcat.xml [hunk 1]
```diff
@@ -219,6 +270,48 @@ $Id$
       its values, which are used as direct references to data categories; hence the use of
         <att>datcat</att> in the <gi>symbol</gi> element.</p>
   </exemplum>
+  <exemplum xml:lang="en">
+    <p>The <ident type="class">att.datcat</ident> attributes can be used for any sort of taxonomies. The example below illustrates their usefulness for describing usage domain labels in dictionaries on the example of Morais (REFERENCE). </p>
```
Comment (ebeshero) [inline P5/Source/Specs/att.datcat.xml]: @bansp Council is reviewing this at Paderborn, and noticed that we're missing a reference here--presumably something to add to BIB, too. Can you add these?

Comment (bansp) [reply P5/Source/Specs/att.datcat.xml]: Thanks, @ebeshero . I left that for the Lex-0 colleagues to add, and with at least @ttasovac and (I hope) @laurentromary present in Paderborn, this is going to be a quick job.

Comment (bansp) [reply P5/Source/Specs/att.datcat.xml]: @ebeshero Please kindly review the most recent edit. I've made it online, so without any checks beyond those offered by Github. I hope hi/@rend="it" was the right markup to use.

Comment (bansp) [reply P5/Source/Specs/att.datcat.xml]: I've backported the edit to the LingSIG repo and the result is going to be soon shown under 
https://jenkins-paderborn.tei-c.org/view/LingSIG/job/TEIP5-LingSIG-tests/lastSuccessfulBuild/artifact/P5/release/doc/tei-p5-doc/en/html/ref-att.datcat.html

File: P5/Source/Specs/att.datcat.xml [hunk 2]
```diff
@@ -219,6 +270,52 @@ $Id$
       its values, which are used as direct references to data categories; hence the use of
         <att>datcat</att> in the <gi>symbol</gi> element.</p>
   </exemplum>
+  <exemplum xml:lang="en">
+    <p>The <ident type="class">att.datcat</ident> attributes can be used for any sort of taxonomies.
+      The example below illustrates their usefulness for describing usage domain labels in
+      dictionaries on the example of the <hi rend="it">Diccionario da Lingua Portugueza</hi> by
```
Comment (ebeshero) [inline P5/Source/Specs/att.datcat.xml]: @bansp This `<hi rend='it`>` should probably be `<title level='m'>`, yes?

**Review by ebeshero — CHANGES_REQUESTED** [review]
@bansp Can you change that `<hi rend="it">` to `<title level='m'>`? (Or is there a reason you didn't use it?)

File: P5/Source/Specs/att.datcat.xml (hunk 2 above)
Comment (bansp) [reply P5/Source/Specs/att.datcat.xml]: By all means. Thanks, Elisa and Hugh, for having a look at it. Gonna change the markup right away,

**Review by ebeshero — APPROVED** [review]
@bansp Reviewing the whole package now--everything looks good to me! I think this is ready to merge.

**Comment by bansp** [comment]
Weeee, thanks for the nice news, Elisa! :-)

**Comment by ebeshero** [comment]
@bansp I'm sorry it took us so long! But perhaps at last the epic day of merging this branch has finally arrived! :-)

**Review by HelenaSabel — APPROVED** [review]

**Comment by bansp** [comment]
Many thanks to everyone involved. It's a nice step forward :-)

**Pull request merged**